var/spack/repos/builtin/packages/umpire/package.py

```diff
@@ -88,7 +90,11 @@ def cmake_args(self):
         options.append('-DBUILD_SHARED_LIBS={0}'.format(
             'On' if '+shared' in spec else 'Off'))
 
+        options.append('-DENABLE_BENCHMARKS={0}'.format(
+            'On' if 'tests=benchmarks' in spec else 'Off'))
+
         options.append('-DENABLE_TESTS={0}'.format(
-            'On' if self.run_tests else 'Off'))
+            'On' if 'tests=basic' in spec
+            or 'tests=benchmarks' in spec else 'Off'))
```

Comment:
To avoid enumerating all the possibilities:
```
options.append(self.define('ENABLE_TESTS',  self.spec.variants['tests'].value != 'none'))
```

Reply:
Just rebased to capture this while preserving consistency with syntax used in setting the other options.